docs/es/docs/advanced/additional-responses.md

```diff
@@ -0,0 +1,257 @@
+# Respuestas adicionales en OpenAPI
+
+/// warning
+
+Este es un tema bastante avanzado.
+
+Si apenas estas empezando con **FastAPI**, Puede que no necesites esto.
+
+///
+
+
+Tú puedes declarar respuestas adicionales, que contengan códigos de estado, tipos de medio, descripciones y varias cosas adicionales.
+
+Ten en cuenta que estas respuestas adicionales se incluirán en el esquema OpenAPI, por lo que también aparecerán en la documentación de la API.
+
+Pero para que esas respuestas adicionales funcionen tienes que asegurarte de que devuelves un `Response` como un objeto`JSONResponse` directamente, el cual debe incluir código de estado y contenido.
+
+## Respuesta Adicional con un `model`
```

Comment:
```suggestion
## Respuesta Adicional con `model`
```
o
```suggestion
## Respuesta Adicional usando `model`
```

Reply:
I think I'll go for the second option thx